d3a_api_client/commands.py

```diff
@@ -0,0 +1,72 @@
+from d3a_api_client.enums import Commands, command_enum_to_command_name
+
+
+class ClientCommandBuffer:
+
+    def __init__(self, ):
+        self._commands_buffer = []
+
+    def offer_energy(self, area_uuid, energy, price):
+        area_uuid = area_uuid
+        args = {energy, price}
+        action = Commands.OFFER
+        self.execute(area_uuid, action, args)
+
+    def offer_energy_rate(self, area_uuid, energy, rate):
+        area_uuid = area_uuid
+        args = {energy, rate}
+        action = Commands.OFFER
+        self.execute(area_uuid, action, args)
+
+    def bid_energy(self, area_uuid, energy, rate):
+        area_uuid = area_uuid
+        args = {energy, rate}
+        action = Commands.OFFER
+        self.execute(area_uuid, action, args)
+
+    def delete_offer(self, area_uuid, offer_id):
+        area_uuid = area_uuid
+        args = {"offer_id": offer_id}
+        action = Commands.DELETE_OFFER
+        self.execute(area_uuid, action, args)
+
+    def delete_bid(self, area_uuid, bid_id):
+        area_uuid = area_uuid
+        args = {"bid_id": bid_id}
+        action = Commands.DELETE_OFFER
+        self.execute(area_uuid, action, args)
+
+    def list_offers(self, area_uuid):
+        area_uuid = area_uuid
+        action = Commands.LIST_OFFERS
+        args = {}
+        self.execute(area_uuid, action, args)
+
+    def list_bids(self, area_uuid):
+        area_uuid = area_uuid
+        action = Commands.LIST_BIDS
+        args = {}
+        self.execute(area_uuid, action, args)
+
+    def device_info(self, area_uuid):
+        area_uuid = area_uuid
+        action = Commands.DEVICE_INFO
+        args = {}
+        self.execute(area_uuid, action, args)
+
+    def execute(self, area_uuid, action, args):
```

Comment:
Should be renamed to `_add_to_buffer` or `_store`. This method is used in order to save the pending commands to the buffer, not to execute them